protocol/errorsHelp.json

```diff
@@ -65,13 +65,14 @@
   "80022": "https://faqs.ably.com/error-code-80022-unable-to-find-connection",
   "90001": "https://faqs.ably.com/error-code-90001-channel-operation-failed-invalid-channel-state",
   "90003": "https://faqs.ably.com/error-code-90003-unable-to-recover-channel-messages-expired",
-  "90007": "https://faqs.ably.com/error-code-40006-invalid-connection-id",
+  "90007": "https://faqs.ably.com/error-code-90007",
   "90010": "https://faqs.ably.com/error-code-90010-maximum-number-of-channels-per-connection-exceeded",
   "90021": "https://faqs.ably.com/error-code-90021-max-channel-creation-rate-exceeded",
   "91000": "https://faqs.ably.com/error-code-91000",
   "91001": "https://faqs.ably.com/error-code-91001",
   "91003": "https://faqs.ably.com/error-91003-maximum-member-limit-exceeded",
   "91005": "https://faqs.ably.com/error-code-91005",
+  "91200": "https://faqs.ably.com/error-code-90007",
```

Comment:
Should this be `https://faqs.ably.com/error-code-91200`?

Reply:
91200 was only in ably-java and has been replaced by 90007
https://github.com/ably/ably-java/pull/961/files

Reply:
Ok thanks 👍